GeoHealthCheck/app.py

```diff
@@ -84,6 +94,33 @@ def db_commit():
     return err
 
 
+def start_crons():
+    # Cold start evry cron of evry ressource
+    for resource in Resource.query.all():
+        freq = resource.test_frequency
```

Comment:
Do this via setting a default value for `test_frequency` in Models.py and the migration script. No need to set here.